packages/contracts-bedrock/src/dispute/SuperFaultDisputeGame.sol

```diff
@@ -69,6 +62,9 @@ import { IDisputeGame } from "interfaces/dispute/IDisputeGame.sol";
 /// @title SuperFaultDisputeGame
 /// @notice An implementation of the `IFaultDisputeGame` interface for interop.
 contract SuperFaultDisputeGame is Clone, ISemver {
+    /// @dev Error to prevent initialization a dispute game with an actually valid, invalid state
+    error SuperFaultDisputeGameInvalidRootClaim();
```

Comment:
For my own context; Are you inlining this error into the contract due to code freeze restrictions? If not, would prefer this to be moved to `src/dispute/lib/Errors.sol`

Reply:
I'm inlining it to fit with our[ new errors policy ](https://github.com/ethereum-optimism/design-docs/blob/main/solidity/revert-strings-to-custom-error-migration.md) which I know we still need to actually migrate towards, but figured I might as well start here